setup.py

```diff
@@ -20,9 +24,32 @@
 ]
 
 
+UI_DOWNLOAD_URL = 'https://github.com/ansible/ansible-hub-ui/' + \
+    'releases/latest/download/automation-hub-ui-dist.tar.gz'
+
+
+class BuildPyCommand(setuptools.command.build_py.build_py):
+    """Custom build command."""
+
+    def run(self):
+        print('Downloading UI static files')
+
+        filename, headers = urllib.request.urlretrieve(
+            UI_DOWNLOAD_URL,
+            'automation-hub-ui-dist.tar.gz'
+        )
+
+        print('Extracting ' + filename)
+        tarfile.open(filename).extractall(
+            path='galaxy_ng/app/static/galaxy_ng'
+        )
+
+        setuptools.command.build_py.build_py.run(self)
+
+
 setup(
     name="galaxy-ng",
-    version="0.1.0a1",
+    version=version.get_git_version(),
```

Comment:
I'm not sure that using git for versioning is a good idea, though it's acceptable for now.